### DISCUSSION TITLE
Markdown highlights

### DISCUSSION THREAD

**Discussion by adrian5** [2giosangmitom/nightfall.nvim#53] [Ideas]
Right now code isn't distinct from regular text and quoted text looks very intense in pink (I often have larger segments like this):

![markdown-preview](https://github.com/user-attachments/assets/0365450a-b8c4-4d9d-afa3-7eb5e3477f04)

What do you think about changing [highlights](https://github.com/2giosangmitom/nightfall.nvim/blob/master/lua/nightfall/groups/integrations/treesitter.lua#L120-L136) to something similar to this:

![markdown-preview-2](https://github.com/user-attachments/assets/b2174b27-4690-448b-b465-5b2bd5e3d98d)

The code uses treesitter `@markup.raw [.markdown_inline|.block.markdown]`. I'm thinking good candidates for Nightfall are:

* `pale_yellow` `#f2fa95`
* `peach` `#ffb86c`
* `pink` `#ff6bcb`
* `violet` `#9a86fd`

The quoted text uses treesitter `@markup.quote [.markdown]`. Good candidates could be:

* `foreground` `#e9e9f4`
* `lavender` `#b4befe`
* `light_grey` `#6060a4`
* `sand` `#e5c697`
* `violet` `#9a86fd`

**Comment by 2giosangmitom** [comment]
Send me PR, thank you for your suggestion.

**Reply by adrian5** [reply]
![pink-turquoise](https://github.com/user-attachments/assets/0dadebfc-3b32-4d02-8adc-f084840bc64b)

I think **pink** works great for headings, it's strong but it looks harmonious. What do you think? We could utilize **turquoise** for something else then. For quotes I feel the dark `light_grey` works best. I could also experiment with your `lighten/darken()` functions later on, but this would already be an improvement.

**Reply by 2giosangmitom** [reply]
It's looks better than the older.
